llms/mlx_lm/lora.py

```diff
@@ -95,14 +100,19 @@ def build_parser():
         choices=["lora", "dora", "full"],
         help="Type of fine-tuning to perform: lora, dora, or full.",
     )
-
+    parser.add_argument(
+        "--optimizer",
+        type=str,
+        choices=["adam", "adamw"],
+        default="adam",
```

Comment:
One more thing, I think this needs to default to `None` to get properly read from the config file.

That logic goes:

if arg is None:
  arg = from_config_file

if arg is still None:
  arg = from CONFIG DEFAULTS